airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/base/IntegrationRunnerTest.java

```diff
@@ -275,4 +293,97 @@ void testDestinationConsumerLifecycleFailure() throws Exception {
     }
   }
 
+  @Test
+  void testInterruptOrphanThreadFailure() {
+    System.setIn(new ByteArrayInputStream("{}\n{}".getBytes()));
```

Comment:
what is System.setIn for?

Reply:
this was already used in this test class before, from what I see, you can define the "input" stream of STDIN with this.

So when the consumer starts to read from STDIN, it'll but those airbyte messages